x-pack/plugins/index_lifecycle_management/public/application/sections/edit_policy/components/snapshot_policies/snapshot_policies.tsx

```diff
@@ -0,0 +1,157 @@
+  }));
+
+  const onComboChange = (options: EuiComboBoxOptionOption[]) => {
+    if (options.length > 0) {
+      onChange(options[0].label);
+    } else {
+      onChange('');
+    }
+  };
+
+  const onCreateOption = (newValue: string) => {
+    onChange(newValue);
+  };
+
+  let calloutContent;
+  if (error) {
+    calloutContent = (
+      <Fragment>
+        <EuiSpacer size="m" />
+        <EuiCallOut
+          data-test-subj="policiesErrorCallout"
+          size="s"
+          iconType="help"
+          color="warning"
+          title={
+            <Fragment>
+              <FormattedMessage
+                id="xpack.indexLifecycleMgmt.editPolicy.deletePhase.noPoliciesLoadedTitle"
+                defaultMessage="Couldn't load existing policies."
+              />
+
+              <EuiButtonIcon
+                size="s"
+                color="warning"
+                onClick={sendRequest}
+                iconType="refresh"
+                aria-label={i18n.translate(
+                  'xpack.indexLifecycleMgmt.editPolicy.deletePhase.reloadPoliciesLabel',
+                  {
+                    defaultMessage: 'Reload policies',
+                  }
+                )}
+              />
+            </Fragment>
+          }
+        >
+          <FormattedMessage
+            id="xpack.indexLifecycleMgmt.editPolicy.deletePhase.noPoliciesLoadedMessage"
+            defaultMessage="You still can type in a policy name."
+          />
+        </EuiCallOut>
+      </Fragment>
+    );
+  } else if (data.length === 0) {
+    calloutContent = (
+      <Fragment>
+        <EuiSpacer size="m" />
+        <EuiCallOut
+          data-test-subj="noPoliciesCallout"
+          size="s"
+          iconType="help"
+          color="warning"
+          title={
+            <FormattedMessage
+              id="xpack.indexLifecycleMgmt.editPolicy.deletePhase.noPoliciesCreatedTitle"
+              defaultMessage="No policies"
+            />
+          }
+        >
+          <FormattedMessage
+            id="xpack.indexLifecycleMgmt.editPolicy.deletePhase.noPoliciesCreatedMessage"
+            defaultMessage="You haven't created any snapshot policies yet. You still can type in a policy name."
+          />
+        </EuiCallOut>
+      </Fragment>
+    );
+  } else if (value && !data.includes(value)) {
+    calloutContent = (
+      <Fragment>
+        <EuiSpacer size="m" />
+        <EuiCallOut
+          data-test-subj="customPolicyCallout"
+          size="s"
+          iconType="help"
+          color="warning"
+          title={
+            <FormattedMessage
+              id="xpack.indexLifecycleMgmt.editPolicy.deletePhase.customPolicyTitle"
+              defaultMessage="Your input doesn't match any existing snapshot policies."
+            />
+          }
+        >
+          <FormattedMessage
+            id="xpack.indexLifecycleMgmt.editPolicy.deletePhase.customPolicyMessage"
+            defaultMessage="You can still save this value."
```

Comment:
```suggestion
            defaultMessage="However, you can create a new policy with this name."
```

Reply:
Hi @lockewritesdocs, thank you for the suggestions! I just wanted to clarify the user flow for the "wait for snapshot policy" field. The user can input any value in the combo box, but a new snapshot policy will not be created automatically for them. We want to encourage them to select an existing snapshot policy or if they type in a custom value, I'll add a link (in a later PR) to "Snapshot and Restore" where they can create a new snapshot policy. 
<img width="446" alt="Screenshot 2020-07-08 at 14 25 36" src="https://user-images.githubusercontent.com/6585477/86920122-8789db00-c129-11ea-927a-b55fdcca307f.png">

Reply:
@yuliacech, thank you for the explanation. So we are encouraging users to choose an existing snapshot policy, correct? In that case, perhaps this text aligns better with that strategy, and provides a link for how to create a new policy:

>Policy name not found
>Enter the name of an existing snapshot policy, or [create a new policy](https://www.elastic.co/guide/en/elasticsearch/reference/current/getting-started-snapshot-lifecycle-management.html#slm-gs-create-policy) with this name.

Reply:
@lockewritesdocs yes, I think that is a great wording for this! Thank you 👍